internal/plugins/manifests/v2/plugin.go

```diff
@@ -36,17 +39,18 @@ var (
 // Config configures this plugin, and is saved in the project config file.
 type Config struct{}
 
-// hasPluginConfig returns true if cfg.Plugins contains an exact match for this plugin's key.
-func hasPluginConfig(cfg *config.Config) bool {
-	if !cfg.IsV3() || len(cfg.Plugins) == 0 {
+// hasPluginConfig returns true if cfg.Layout contains an exact match for this plugin's key.
+func hasPluginConfig(cfg config.Config) bool {
+	isV3 := cfg.GetVersion().Compare(cfgv3.Version) >= 0
+	if !isV3 {
 		return false
 	}
```

Comment:
Same as above, not needed this special case because v2 already will return false
```suggestion
```